From d35cfca63390950acbdc2c90ff8734abe23c639c Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Sun, 1 Nov 2020 17:11:52 +0800
Subject: [PATCH 1/6] scx35-common: Update wpa_supplicant.rc for Pie

Now, Android framework handles the configuration locations of the
wpa_supplicant which is also now at /data/vendor/... Change the service
definition accordingly.

Yes, this is pretty much the same CL as with the kanas device tree.
This is not kanas configuration centric patch anyways.

Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
Change-Id: I66c40463157775e8b1860e9231518c080fa6b199
---
 system/etc/init/wpa_supplicant.rc | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/system/etc/init/wpa_supplicant.rc b/system/etc/init/wpa_supplicant.rc
index b2d8664..72c0efe 100644
--- a/system/etc/init/wpa_supplicant.rc
+++ b/system/etc/init/wpa_supplicant.rc
@@ -1,9 +1,10 @@
 service wpa_supplicant /vendor/bin/hw/wpa_supplicant \
-    -iwlan0 -Dnl80211 -c/data/misc/wifi/wpa_supplicant.conf \
-    -I/vendor/etc/wifi/wpa_supplicant_overlay.conf \
+    -O/data/vendor/wifi/wpa/sockets \
     -puse_p2p_group_interface=1use_multi_chan_concurrent=0p2p_device=1 \
-    -e/data/misc/wifi/entropy.bin -g@android:wpa_wlan0
+    -g@android:wpa_wlan0
     class main
-    socket wpa_wlan0 dgram 660 wifi wifi u:object_r:wpa_socket:s0
+    interface android.hardware.wifi.supplicant@1.0::ISupplicant default
+    interface android.hardware.wifi.supplicant@1.1::ISupplicant default
+    socket wpa_wlan0 dgram 660 wifi wifi
     disabled
     oneshot
-- 
2.28.0

